assets/app/scripts/services/limits.js

```diff
@@ -33,14 +33,21 @@ angular.module("openshiftConsole")
       return usageValue(candidate) > usageValue(previous);
     };
 
-    var hideCPUComputeResources = function() {
-      return LIMIT_REQUEST_OVERRIDES &&
-             LIMIT_REQUEST_OVERRIDES.limitCPUToMemoryPercent &&
-             LIMIT_REQUEST_OVERRIDES.cpuRequestToLimitPercent;
+    // Check if compute resources overrides are enabled for this project.
+    var limitRequestOverridesEnabled = function(project) {
+      // If config is absent, overrides are disabled.
+      if (!LIMIT_REQUEST_OVERRIDES) {
+        return false;
+      }
+
+      // If the project resource override annotation is unset or is set to "true", overrides are enabled.
+      var projectOverrideAnnotation =
+        $filter('annotation')(project, 'quota.openshift.io/cluster-resource-override-enabled');
+      return !projectOverrideAnnotation || projectOverrideAnnotation === 'true';
```

Comment:
@sosiouxme please check this behavior: overrides are enabled if config is set AND project annotation is missing or set to the string `"true"`.